src/FSharpVSPowerTools.Logic/QuickInfoMargin.fs

```diff
@@ -136,7 +136,7 @@ type QuickInfoMargin (textDocument: ITextDocument,
             |> Async.StartInThreadPoolSafe
         | None -> updateQuickInfo (None, None)
 
-    let docEventListener = new DocumentEventListener ([ViewChange.layoutEvent view; ViewChange.caretEvent view], 1000us, updateAtCaretPosition)
+    let docEventListener = new DocumentEventListener ([ViewChange.layoutEvent view; ViewChange.caretEvent view], 200us, updateAtCaretPosition)
```

Comment:
OK, I see the comment in the PR.  Does this increase the frequency of doing updates to the QuickInfo margin?  will that result in more FCS requests?  (Currently I'm seeing cases where the FCS request queue grows to be very long very quickly - and the requests aren't yet being cancelled when new requests come in - so changes that increase the number of FCS requests have to be done with care)